R/Rules-methods.R

```diff
@@ -865,14 +865,10 @@ setMethod(
       next_dose = next_dose_drt
     )
 
-    dose_grid_range <- c(
-      data@doseGrid[ifelse(data@placebo && data@nGrid >= 2, 2, 1)],
-      data@doseGrid[data@nGrid]
-    )
-    if (!h_in_range(dose_target_drt, range = dose_grid_range, bounds_closed = TRUE) && !SIM) {
+    if (!h_in_range(dose_target_drt, range = h_dose_grid_range(data), bounds_closed = TRUE) && !SIM) {
```

Comment:
unrelated but: the `SIM` variable does not follow conventions, can we use something else? e.g. `do_sim` or so?

Reply:
It seems that this would need to be changed in a few places - so I will create a new issue for this.